### ISSUE TITLE
PRIORITY:  History & Reporting

### ISSUE THREAD

**Issue by sa-sturner** [pltarr/sa-database-testing#95]
@putn1nr 
Hi Nicole, 

We have to have history and all reporting before we can roll out.

**Comment by putn1nr** [comment]
@sa-sturner Hi Susie. Attached is the Logbook Report scope (both the interface/application report and the export report out). Would you please take a moment to review and let me know if you'd like me to make any adjustments. Thanks!

[Log Book Report Export 3-25-19.xlsx](https://github.com/pltarr/sa-database-testing/files/3005070/Log.Book.Report.Export.3-25-19.xlsx)
[Opportunity Status Logic-3-20-19.docx](https://github.com/pltarr/sa-database-testing/files/3005071/Opportunity.Status.Logic-3-20-19.docx)

**Comment by sa-sturner** [comment]
@putn1nr 
Hi Nicole, 
The opportunity status logic I responded on another issue. 

On the log book report export I do have a couple of questions and adjustments.  
Adjustments:  
1.  We do not need to include Date Registered or Date Approved for the Angel or the Service Member.  
2.  We do not need Service Member Address 1, Address 2, City, State, Zip or Phone Number.  
3.  We do need to know what items are being reporting for sewing and crafting (blankets, caps, etc.). 

Questions
1.  Is this going to pull into a spreadsheet?
2.  Are we able to see details mode (what is currently   shown) as well as summary mode (where Britt Smith is combined into one line).
3.  What about extra support reporting? Would it be on here as well? 
4.  Are rows A2-A5 are sort features? Date Range, Angel Name, Service Member Name and Program?
5.  If Angel Name and Service Member name is a filter is it a find feature or do we have to have the exact name?

**Comment by putn1nr** [comment]
@sa-sturner Hi Susie. I have attached the updated scopes (both the interface report and the excel report).

Below are the answers to your questions:

1. Is this going to pull into a spreadsheet? **Yes, upon clicking the "Export to Excel" link on the Logbook Report (on the application), the report would generate and download in Excel.**

2. Are we able to see details mode (what is currently shown) as well as summary mode (where Britt Smith is combined into one line). **As of right now, the Logbook Report is a detail report where we would show every record/activity recorded for a Team Angel. We could produce 2 reports; Logbook Report - Detail and Logbook Report - Summary. Would you like us to scope the Summary?**

3. What about extra support reporting? Would it be on here as well? **Additional Support (records entered via the Additional Support Form on the Team Angel Portal) would show in the Logbook Report. Since Additional Activity isn't directly tied to a Service Member, in the "Service Member" column of the Report,, we would show = Additional Support. That way, you would know that the activity is an extra/additional support record.**

4. Are rows A2-A5 are sort features? Date Range, Angel Name, Service Member Name and Program? **The Date Range, Angel Name, Service Member Name, and Program would all be filters that you selected on the application first. You'd be able to generate the report for a select period of time (one day, a week, month, etc.) by a specific Service Member or Team Angel, or by Program. At mimimum, you'd need to filter by a period of time.**

5. If Angel Name and Service Member name is a filter is it a find feature or do we have to have the exact name? **The Angel Name and Service Member Name filters could either be type-ahead fields or dropdowns. What would you prefer?**

[Log Book Report Export 3-29-19 V2.xlsx](https://github.com/pltarr/sa-database-testing/files/3023977/Log.Book.Report.Export.3-29-19.V2.xlsx)
[Logbook Report V3 3-29-19.pdf](https://github.com/pltarr/sa-database-testing/files/3023978/Logbook.Report.V3.3-29-19.pdf)
